gulpfile.js

```diff
@@ -193,36 +196,19 @@ function removeItem(platforms, item) {
 }
 
 function getRunDebugAppCommand(arch) {
-
-    let command;
-
     switch (arch) {
     case 'osx64':
-        const pkgName = `${pkg.name}.app`;
-        command = `open ${path.join(DEBUG_DIR, pkg.name, arch, pkgName)}`;
-
-        break;
-
+        return `open ${path.join(DEBUG_DIR, pkg.name, arch, pkg.name)}.app`;
     case 'linux64':
     case 'linux32':
     case 'armv7':
-        command = path.join(DEBUG_DIR, pkg.name, arch, pkg.name);
-
-        break;
-
+        return path.join(DEBUG_DIR, pkg.name, arch, pkg.name);
     case 'win32':
     case 'win64':
-        command = path.join(DEBUG_DIR, pkg.name, arch, `${pkg.name}.exe`);
-
-        break;
-
+        return path.join(DEBUG_DIR, pkg.name, arch, pkg.name + '.exe');
     default:
-        command =  '';
-
-        break;
+        return '';
     }
-
-    return command;
```

Comment:
All the changes of `getRunDebugAppCommand()` are not needed. In your first versions Sonar complained about, but the problem was fixed in master and after your last rebase this changes are not needed.
If you think your code is better than the actual for this method, please open another PR for it. This changes have nothing to do with this PR and are better off to simplify a little.